splinterd/Cargo.toml

```diff
@@ -96,6 +96,10 @@ admin-service-event-store = [
   "splinter/admin-service-event-store",
   "splinter/admin-service"
 ]
+admin-service-event-store-diesel = [
+  "admin-service-event-store",
+  "splinter/admin-service-event-store-diesel"
```

Comment:
I don’t think this needs to be a separate feature in splinterd, the diesel feature should just be added to the “admin-service-event-store” feature (since splinterd requires a DB now)

Reply:
I changed the `admin-service-event-store` feature here to pull in `splinter/admin-service-event-store-diesel`